Neos.ContentRepository.BehavioralTests/Classes/Command/PerformanceMeasurementService.php

```diff
@@ -154,9 +153,10 @@ private function createHierarchy(
 
     public function forkContentStream(): void
     {
-        $this->contentRepository->handle(ForkContentStream::create(
-            ContentStreamId::create(),
-            $this->contentStreamId,
-        ));
+        throw new \BadMethodCallException('not implemented');
+        // $this->contentRepository->handle(ForkContentStream::create(
+        //     ContentStreamId::create(),
+        //     $this->contentStreamId,
+        // ));
```

Comment:
?

Reply:
well remove the `ForkContentStream` in the followup and i wanted to already get rid of those usages ... the performance tests are not run in ci and must be reworked after all the adjustments of the last year anyway.

Reply:
We should / have to throw away everything here then though because it doesn't make sense anymore without it. Fine by me, I would try to replace it with a bench scenario if we want to.

Reply:
removed adjustments here